DEFAULT_CONFIG.json5

```diff
@@ -471,14 +471,14 @@
           /// then amount of memory being allocated for each queue is SIZE_XXX * LINK_MTU.
           /// If qos is false, then only the DATA priority will be allocated.
           size: {
-            control: 2,
-            real_time: 2,
-            interactive_high: 2,
-            interactive_low: 2,
+            control: 1,
+            real_time: 1,
+            interactive_high: 1,
+            interactive_low: 1,
             data_high: 2,
-            data: 2,
-            data_low: 2,
-            background: 2,
+            data: 4,
+            data_low: 4,
```

Comment:
The numbers are not coherent with `commons/zenoh-config/src/defaults.rs`.

Reply:
You're right, good catch !